_data/site_menu.yml

```diff
@@ -1,77 +1,56 @@
 ---
 
-installation:
-  title: Installation
+installing:
+  title: Installing
   children:
-    - title: Amazon Web Services (AWS)
-      path: "/docs/reference/latest/installing_on_aws/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Amazon Web Services (AWS)"
-    - title: Kubernetes
-      path: "/docs/reference/latest/installing_on_kubernetes/index.html"
-      desc: "How to install and configure ManageIQ on Kubernetes"
-    - title: IBM Cloud
-      path: "/docs/reference/latest/installing_on_ibm_cloud/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on IBM Cloud"
-    - title: Microsoft Azure
-      path: "/docs/reference/latest/installing_on_microsoft_azure/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft Azure"
-    - title: Google Compute Engine
-      path: "/docs/reference/latest/installing_on_google_compute_engine/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Google Compute Engine"
-    - title: VMware vSphere
-      path: "/docs/reference/latest/installing_on_vmware_vsphere/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on VMware vSphere"
-    - title: Red Hat Enterprise OpenStack Platform
-      path: "/docs/reference/latest/installing_on_red_hat_enterprise_linux_openstack_platform/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Red Hat OpenStack Platform"
-    - title: Red Hat Enterprise Virtualization
-      path: "/docs/reference/latest/installing_on_red_hat_virtualization/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Red Hat Enterprise Virtualization"
-    - title: Microsoft System Center Virtual Machine Manager (SCVMM)
-      path: "/docs/reference/latest/installing_on_scvmm/index.html"
-      desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
-
-appliances:
-  title: Appliances
+    - title: Containerized (podified)
+        children:
+           - title: Kubernetes
+             path: "/docs/reference/latest/installing_on_kubernetes/index.html"
+             desc: "How to install and configure ManageIQ on Kubernetes"
+    - title: Appliance
+        children:
+           - title: Amazon Web Services (AWS)
+             path: "/docs/reference/latest/installing_on_aws/index.html"
+             desc: "How to install and configure the ManageIQ Management Engine appliance on Amazon Web Services (AWS)"
```

Comment:
Were we going to remove the `Management Engine` part throughout or is that for a followup PR?

Reply:
It should just be ManageIQ for the open source product.